src/test/java/httpserver/HttpResponderTest.java

```diff
@@ -48,12 +56,27 @@ public boolean isClosed() {
     private class SimpleHttpResponsePresenter implements Presentable {
         private String response;
 
-        public SimpleHttpResponsePresenter(String response) {
-            this.response = response;
+        @Override
+        public String present(HttpResponse response) {
+            return response.responseLine + System.lineSeparator();
         }
+    }
+
+    private class SimpleHttpParser implements Parseable {
 
         @Override
-        public String present() {
+        public String parse(String request) {
+            String[] splitRequest = request.split(" ");
+            return splitRequest[1];
+        }
+    }
+
+    private class SimpleHttpController implements Controller{
+        public HttpResponse control(String request) {
+            HttpResponse response = new HttpResponse();
+            if(request.equals("/hello_world")) {
+                response.responseLine += "200 Ok";
+            }
```

Comment:
Do we have another test that uses this conditional? If not, do we need it?

Reply:
We don't. I modified.